client/themes/default.css

```diff
@@ -15,6 +15,12 @@
 	box-shadow: 0 0 25px rgb(0 0 0 / 50%);
 }
 
+@media screen and (orientation: portrait) {
+	.window {
+		box-shadow: 0 0 0;
```

Comment:
why are you messing with the shadows here?

Reply:
It was an attempt to make it look a bit more at home on iOS. Here it is with and without these rules, respectively:

<img width="244" alt="Channel in Portrait" src="https://github.com/user-attachments/assets/b31a1215-e0d7-4f40-89eb-ca7a26b0a4d2">
<img width="244" alt="Channel Portrait (Without Chrome)" src="https://github.com/user-attachments/assets/eaf658ad-8161-4e1b-9876-7cb3c01880d1">

But probably better left to CSS overrides or a separate theme maybe, so I left it out. I'll update the screenshots in the merge request to reflect the new appearance.

Reply:
I definitely think that the background should be the same color as the text input box, perhaps there is a different way to enforce that?